src/pages/Business.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+import Layout from "../components/layout";
+import SEO from "../components/seo";
+
+const Business = () => (
+    <Layout>
+        <SEO title="Business vocabulary" />
+        <h1>Life Engine vocabulary - Business</h1>
+            <a name={"RegistrationDate"}><h2>RegistrationDate</h2></a>
```

Comment:
everything else is snake case here, why is this starting with a capital letter, and why does  `RegistrationDate` not have spaces in between the words inside the `<h2>`?